modules/ROOT/nav.adoc

```diff
@@ -16,8 +16,10 @@
 **** xref:rails-cloud.adoc[Ruby on Rails]
 *** Self-hosted
 **** React
-***** xref:react-pm.adoc[Using a package manager]
-***** xref:react-zip.adoc[Using a .zip package]
+***** xref:react-pm-host.adoc[Using a package manager with hosting]
+***** xref:react-pm-bundle.adoc[Using a package manager with bundling]
+***** xref:react-zip-host.adoc[Using a .zip package with hosting]
+***** xref:react-zip-bundle.adoc[Using a .zip package with bundling]
```

Comment:
This reads a little strange but I don't really have any suggestions. The best I can think of is swapping the hosting/bundling to be first, e.g. `Hosting using a package manager` and `Bundling using a package manager`

Reply:
The problem is then it starts different to all the other headings that say just "Using a package manager" and "Using a .zip package".